api/serverless.yml

```diff
@@ -219,6 +191,49 @@ functions:
                   path: ${self:custom.versions.v1}/publications/research-topics
                   method: GET
                   cors: true
+    getPublicationTopics:
+        handler: src/components/publication/routes.getPublicationTopics
+        events:
+            - http:
+                  path: ${self:custom.versions.v1}/publications/{id}/topics
+                  method: GET
+                  cors: true
+    # Publication Versions
+    getPublicationVersion:
+        handler: src/components/publicationVersion/routes.get
+        events:
+            - http:
+                  path: ${self:custom.versions.v1}/publications/{id}/versions/{version} # {version} can be a version id, number or "latest"
+                  method: GET
+                  cors: true
+    getIndexedPublicationVersions:
+        handler: src/components/publicationVersion/routes.getAll
+        events:
+            - http:
+                  path: ${self:custom.versions.v1}/versions
```

Comment:
I would prefer to keep the publication versions endpoint as /publicationVersions since it's clearer what resource it's about. To an API user looking at this for the first time they may wonder "a version of what?" if they only see the URL. It also means we have paths like /users/{id}/versions, which makes it look like it's about versions of the user. We may also have other entities with versions one day.

Reply:
Renamed "/versions" endpoints to "/publication-versions" and updated tests